docs/configuration.md

```diff
@@ -32,8 +32,8 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `HEDERA_NETWORK`           | ""             | Which network to connect to. Automatically populates the main node & mirror node endpoints. Can be `previewnet`, `testnet`, `mainnet` or a map of network IPs -> node accountIds e.g. `{"127.0.0.1:50211":"0.0.3"}` |
 | `INPUT_SIZE_LIMIT`         | "1mb"          | The [koa-jsonrpc](https://github.com/Bitclimb/koa-jsonrpc) maximum size allowed for requests                                                                                                                        |
 | `MAX_BLOCK_RANGE`          | ""             | The maximum block number greater than the mirror node's latest block to query for                                  
-| `OPERATOR_ID_MAIN`         | ""             | Operator account ID used to pay for transactions.                                                                                                                                                                                |
-| `OPERATOR_KEY_MAIN`        | ""             | Operator private key used to sign transactions in hex encoded DER format.                                                                                                                                           |
+| `OPERATOR_ID_MAIN`         | ""             | Operator account ID used to pay for transactions. In `S.R.N` format. |
+| `OPERATOR_KEY_MAIN`        | ""             | Operator private key used to sign transactions in hex encoded DER format. This may be either an ED22519 private key or an ECDSA private key. The private key must be associated with/ used to derive `OPERATOR_ID_MAIN`. |
```

Comment:
```suggestion
| `OPERATOR_KEY_MAIN`        | ""             | Operator private key used to sign transactions in hex encoded DER format. This may be either an ED22519 private key or an ECDSA private key. The private key must be associated with/used to derive `OPERATOR_ID_MAIN`. |
```